tests/update/slowroll_repos.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add slowroll repos in the system
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
```

Comment:
@foursixnine IMHO this could/should be core-team, ok?

Reply:
@b10n1k Can you clarify? I don't see a change, or comment here

Reply:
As the author i used myself. But this can change later when @foursixnine clarify the @okurz 's question

Reply:
Fine, let's keep as is for now

Reply:
@okurz indeed! @b10n1k I'm fine with this being en qe-core, I think we can somehow merge the code, but since Yogesh left, we haven't been able to help here again.